learning/tour-of-beam/frontend/lib/pages/tour/screen.dart

```diff
@@ -0,0 +1,358 @@
+  const _NarrowTour();
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
+      child: Column(
+        children: [
+          Row(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: const [
+              _ContentTree(),
+              Expanded(child: _Content()),
+            ],
+          ),
+          DecoratedBox(
+            decoration: BoxDecoration(
+              border: Border(
+                top: BorderSide(color: ThemeColors.of(context).divider),
+              ),
+            ),
+            child: const _Playground(),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _ContentTree extends StatelessWidget {
+  const _ContentTree();
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 250,
+      padding: const EdgeInsets.symmetric(horizontal: BeamSizes.size12),
+      child: SingleChildScrollView(
+        child: Column(
+          children: [
+            const _ContentTreeTitle(),
+            ...[
+              'Core Transforms',
+              'Common Transforms',
+            ].map((e) => _Module(module: e)).toList(),
+            const SizedBox(height: BeamSizes.size12),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _Module extends StatelessWidget {
+  final String module;
+  const _Module({required this.module});
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        _ModuleTitle(title: module),
+        ...['Map', 'Combine'].map((e) => _Groups(group: e)).toList(),
+        const BeamDivider(
+          margin: EdgeInsets.symmetric(vertical: BeamSizes.size10),
+        ),
+      ],
+    );
+  }
+}
+
+class _ContentTreeTitle extends StatelessWidget {
+  const _ContentTreeTitle();
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: BeamSizes.size12),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          Text(
+            'pages.tour.summaryTitle',
+            style: Theme.of(context).textTheme.headlineLarge,
+          ).tr(),
+        ],
+      ),
+    );
+  }
+}
+
+class _ModuleTitle extends StatelessWidget {
+  final String title;
+  const _ModuleTitle({required this.title});
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(bottom: BeamSizes.size6),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          Text(
+            title,
+            style: Theme.of(context).textTheme.headlineMedium,
+          ),
+          const Padding(
+            padding: EdgeInsets.only(right: BeamSizes.size4),
+            child: ComplexityWidget(complexity: Complexity.basic),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _Groups extends StatelessWidget {
+  final String group;
```

Comment:
It should either take `groups` or be `_Group`.

Reply:
Done.